bin/dry-run.js

```diff
@@ -13,6 +14,13 @@ import { ethers } from 'ethers'
 import pg from 'pg'
 import { RoundData } from '../lib/round.js'
 
+Sentry.init({
+  dsn: 'https://d0651617f9690c7e9421ab9c949d67a4@o1408530.ingest.sentry.io/4505906069766144',
+  environment: 'dry-run',
```

Comment:
I propose to set the environment to "CI" when running on GitHub Actions:

```suggestion
  environment: process.env.SENTRY_ENVIRONMENT || 'dry-run',
```

See also:
https://github.com/filecoin-station/spark-evaluate/blob/b1280f727f3e5bd29c7257fdbdeba7c6eedf077e/bin/spark-evaluate.js#L15-L18